public_html/app/Libraries/Core.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace VFramework\Libraries;
+
+use VFramework\Controllers\Posts;
+use VFramework\Models\User;
+use VFramework\Tools\Validator;
+
+/**
+ * Class Core
+ * @package VFramework\Libraries
+ */
+class Core
+{
+    public const CONTROLLERS_DIR = '/app/Controllers/';
+    protected $currentController = 'Pages';
+    protected $currentMethod = 'index';
+    protected $params = [];
+
+    public function __construct()
+    {
+        $url = $this->getUrl();
+        $urlShifted = array_shift($url);
+        // Look in Controllers for first value of URL
+        if (file_exists(ROOT_DIR . self::CONTROLLERS_DIR . ucwords($url[0]) . '.php')) {
```

Comment:
ROOT_DIR . self::CONTROLLERS_DIR . ucwords($url[0]) . '.php' - move to function getController($var) {}
this if statement is not readable.